docs/DesignDocs/SGX_QuoteVerify_Integration.md

```diff
@@ -1,30 +1,33 @@
-Attestation: OE SDK Integration with Intel® SGX DCAP Quote Verification Library for SGX Evidence verification
+Attestation: OE SDK Integration with Intel® SGX DCAP Quote Verification Library for SGX Evidence Verification
 ====
 
-This design document proposes an extension of the OE SDK implementation
+Note: please format the text lines to be within 80 characters if possible.
+
+This design document proposes an update of the OE SDK implementation
 for integration with the Intel® SGX Data Center Attestation Primitives (DCAP) Quote Verification Library (QVL), for support of
 evidence verification in ECDSA-p256 formats.
 
 # Motivation
 
-Existing OE SDK has it's own logic for SGX ECDSA quote verification.
-As implemented in code file `common/sgx/quote.c`, verification code
-has complex logic for SGX cert chain verification and Enclave Identity
+Existing OE SDK implements the functionality for SGX ECDSA quote verification.
+As implemented in code file `common/sgx/quote.c`, the verification code
+performs SGX cert chain verification and Enclave Identity
 verification, including x509 parser, cert revocation checking, JSON and etc.
-Also it would use some 3rd party codes, such as mbedtls.
+Also it would use some 3rd-party code, such as mbedtls.
 
+Howevere, there are a few areas that could be improved:
 - Current SGX ECDSA quote verification logic in `common/sgx/quote.c`
-  is NOT a complete implementation, e.g.
+  is not a complete implementation, e.g.
```

Comment:
Is this check truly incomplete? which other fields must be check for a complete verification?